WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -70,6 +89,12 @@ struct TotalsView: View {
         }
         .onChange(of: shouldShowTotalsFields, perform: hideTotalsFieldsWithDelay)
         .geometryGroupIfSupported()
+        .fullScreenCover(isPresented: $shouldShowCollectCashPayment) {
+            if case .loaded(let total) = posModel.orderState {
+                PointOfSaleCollectCashView(isVisible: $shouldShowCollectCashPayment,
```

Comment:
Ah good catch, that was my initial approach but didn’t work correctly as we were pushing navigation into the stack, dismissing the whole POS, but since we’ve moved to a modal presentation instead it's much simpler to use dismiss. Changed on https://github.com/woocommerce/woocommerce-ios/pull/14724/commits/8d723fa9b9002f3d500150efc66990a917d2925c